core/src/database/wrapper.rs

```diff
@@ -24,7 +24,7 @@ pub struct EVMAddressDB(pub EVMAddress);
 #[derive(Serialize, Deserialize)]
 pub struct TxidDB(pub Txid);
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, sqlx::FromRow)]
```

Comment:
This seems like a column type, `FromRow` should be implemented for a single item tuple like this `(SignatureDb,)`.

Reply:
cc: @ozankaymak

Reply:
I mean it works so maybe later we can change this after we are done with all this? Because I think in general we need to change the database module.